src/FSharp.Data.GraphQL.Samples.GiraffeServer/Schema.fs

```diff
@@ -158,8 +158,8 @@ module Schema =
             isTypeOf = (fun o -> o :? Droid),
             fieldsFn = fun () ->
             [
-                Define.Field("id", String, "The id of the droid.", fun _ d -> d.Id)
-                Define.Field("name", Nullable String, "The name of the Droid.", fun _ d -> d.Name)
+                Define.Field("id", String, "The id of the droid.", fun _ (d : Droid) -> d.Id)
```

Comment:
Unfortunately, creating an overload of `Define.Field` that asks no description, but arguments made this change necessary for it to compile.